services/memory_service.py

```
@@ -79,8 +124,20 @@ def get_by_kind(kinds) -> dict:
     return memory_itens
 
 
+@has_permission(Permission.WRITE_CUSTOM_FORMS)
+def save_custom_form(id: int | None, value, user_context: User):
```

Ruff (0.15.9)

[error] Function argument `id` is shadowing a Python builtin

(A002)